### PR TITLE
filer fix

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5109

*Description of changes:*
 - get name request from filing_json
 - avoid sentry from logging wrong info

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py [hunk 1]
```diff
@@ -28,24 +28,25 @@ def consume_nr(business: Business, filing: Filing, nr_num_path='/filing/incorpor
     """Update the nr to a consumed state."""
     try:
         nr_num = get_str(filing.filing_json, nr_num_path)
-        bootstrap = RegistrationBootstrap.find_by_identifier(filing.temp_reg)
-        namex_svc_url = current_app.config.get('NAMEX_API')
-        token = AccountService.get_bearer_token()
+        if nr_num:
```
Comment (severinbeauvais) [inline queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:31]: What does it mean if the NR Num is not found?

Comment (severinbeauvais) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:31]: Under what conditions would this happen?

Comment (vysakh-menon-aot) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:31]: for IA this function is called in both cases (numbered and named), so for numbered we have to skip this call

Comment (severinbeauvais) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/name_request.py:31]: Thanks.
Please add code comment explaining this.

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**